### ISSUE TITLE
metadata mapping by sample name

### ISSUE THREAD

**Issue by nick-youngblut** [meringlab/FlashWeave.jl#26]
Although not explicit in the README, it appears that the metadata table must have the same sample order as the `sample x feature` table.  For the sake of users who want to make sure that the correct samples are mapped to the correct metadata, it would be helpful to allow users to include a `Sample` column in the metadata table that will map the metadata to the correct sample in the `sample x feature` table.

**Comment by jtackm** [comment]
True, this should definitely be added to the README (I'm actually surprised it isn't). I was pondering to allow user-provided index columns, but didn't get around to implement it yet because there are a number of special cases to consider. Anyways, I agree it would be a good convenience feature.

**Comment by nick-youngblut** [comment]
Great! It would also be a good sanity check to make sure that the user has actually correctly ordered both tables.

**Comment by Dalcon2704** [comment]
Since there is still no information about this problem in the README yet. The problem have been fixed or you have to still sort the metadata?

**Comment by jtackm** [comment]
Yes, metadata still has to be aligned between OTU table and meta data table. I added a statement to the README now.
